docs/src/release_notes/ghaf-24.12.3.md

```diff
@@ -0,0 +1,78 @@
+<!--
+    Copyright 2022-2025 TII (SSRC) and the Ghaf contributors
+    SPDX-License-Identifier: CC-BY-SA-4.0
+-->
+
+# Release ghaf-24.12.3
+
+This is a bi-weekly release with a major update to JetPack 6.2 for NVIDIA platforms.
+
+
+## Release Tag
+
+<https://github.com/tiiuae/ghaf/releases/tag/ghaf-24.12.3>
+
+
+## Supported Hardware
+
+The following target hardware is supported by this release:
+
+* NVIDIA Jetson AGX Orin
+* NVIDIA Jetson Orin NX
+* Generic x86 (PC)—build support only
+* Lenovo ThinkPad X1 Carbon Gen 10, 11, 12
+* NXP i.MX 8M Plus—build support only
+
```

Comment:
you can add that the dell HW is also supported. they are the dell latitude 7330 and 7230 https://github.com/tiiuae/ghaf/blob/main/targets/laptop/flake-module.nix#L54-L71